documentation/samplefiles/readme.typescript.md

```diff
@@ -3,11 +3,12 @@
 These settings apply only when `--typescript` is specified on the command line.
 Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
 
-```yaml $(typescript)
+``` yaml $(typescript)
 typescript:
   azure-arm: true
-  package-name: "YourServiceName"
-  output-folder: "$(typescript-sdks-folder)/packages/YourServiceName"
+  package-name: "@azure/arm-[[ServiceName]]"
+  output-folder: "$(typescript-sdks-folder)/[[ServiceName]]/arm-[[ServiceName]]"
```

Comment:
The output folder should be "$(typescript-sdks-folder)/sdk/[[ServiceName]]/arm-[[ServiceName]]" missing the '/sdk/' here.